topics/tutorial-first-steps-with-kotlin-rpc.topic

```diff
@@ -0,0 +1,600 @@
+                        to help transfer information over the network, then the types used in parameters must be marked with the
+                        <code>Serializable</code> annotation.
+                    </p>
+                </step>
+            </procedure>
+        </chapter>
+        <chapter title="Implement the client" id="client-implementation">
+            <procedure id="client-impl-procedure">
+                <step>
+                    In the client project, navigate to the
+                    <path>Main.kt</path>
+                    file and add the following implementation:
+                    <code-block lang="kotlin"
+                                src="snippets/tutorial-kotlin-rpc-client/src/main/kotlin/Main.kt"
+                                include-lines="1-12,14-35,37-38,58-60"
+                    />
+                </step>
+            </procedure>
+            <p>
+                You only need 25 lines to prepare for and then execute an RPC call. Obviously, there is a lot going on,
+                so let’s break the code down into sections.
+            </p>
+            <p>
+                The <code>kotlinx.rpc</code> library uses the <a href="client-create-new-application.topic">Ktor
+                client</a> to host its runtime on the client side. The runtime is not coupled to Ktor, and other choices
+                are possible, but this promotes reuse and makes it easy to integrate <code>kotlinx.rpc</code> into
+                existing KMP applications.
+            </p>
+            <p>
+                Both the Ktor client and Kotlin RPC are built around coroutines, so you use <code>runBlocking</code> to
+                create the initial coroutine, and execute the rest of the client within it:
+            </p>
+            <code-block lang="kotlin"
+                        src="snippets/tutorial-kotlin-rpc-client/src/main/kotlin/Main.kt"
+                        include-lines="15,60"
+            />
+            <tip>
+                Note that <code>runBlocking</code> is designed for spikes and tests, rather than production code.
+            </tip>
+            <p>
+                Next, you create an instance of the Ktor client in the standard way. <code>kotlinx.rpc</code> uses
+                <a href="client-websockets.md">WebSockets</a> to transfer information, so you need to ensure that the
+                matching plugin is loaded:
+            </p>
+            <code-block lang="kotlin"
+                        src="snippets/tutorial-kotlin-rpc-client/src/main/kotlin/Main.kt"
+                        include-lines="16-18"
+            />
+            <p>
+                Having created this Ktor client, you then create an <code>RPCClient</code> object for invoking remote
+                procedures. You will need to configure the location of the server and the mechanism being used to
+                transfer information:
```

Comment:
From the code example, it is not obvious, which part is an `RPCClient`. I think it would be beneficial to add the type explicitly in the code